generate.js

```diff
@@ -2,6 +2,13 @@ var db = require('./models/');
 const faker = require('faker');
 const mongoose = require('mongoose');
 
+// flush db before seed
+db.Booking.remove({}).exec(function(err, results) {
```

Comment:
booking.remove is a async function. You are saving items into our data base before we know we removed all of them. Refactor so we don't come into this 'silent problem' later on with heavier data